tidb-binlog/pump_client/bench_test.go

```
@@ -21,6 +21,7 @@ import (
 	"time"
 
 	pb "github.com/pingcap/tipb/go-binlog"
+	"github.com/stretchr/testify/assert"
```

```suggestion
	"github.com/stretchr/testify/require"
```

Almost we prefer require to fail fast.

---

sure,i accept it.

---

I have resubmitted the code